pkg/names/names.go

```diff
@@ -24,8 +24,14 @@ import (
 )
 
 const (
-	ServerContainer      = "server"
-	ServerContainerIndex = 0
+	ServerContainer = "server"
+	NMAContainer    = "nma"
+	Index0          = 0
+)
+
+const (
+	NMAContainerIndex = iota
+	ServerContainerIndex
```

Comment:
Can we avoid externalizing these outside of the package? External users should be able to use `GetServerContainerIndex`. So suggest renames of: `nmaContainerIndex` and `serverContainerIndex`.